bower.json

```diff
@@ -0,0 +1,27 @@
+{
+	"name": "Dialer",
+	"version": "1.0.0",
+	"homepage": "https://github.com/fxos/dialer",
+	"authors": [
+		"Doug Sherk <github@sherk.me>"
+	],
+	"description": "Replaceable Dialer for Firefox OS.",
+	"main": "app.js",
+	"license": "MIT",
+	"private": true,
+	"ignore": [
+		"**/.*",
+		"node_modules",
+		"bower_components",
+		"app/components",
+		"test",
+		"tests"
+	],
+	"dependencies": {
+		"fxos-mvc": "git@github.com:fxos/mvc.git",
+		"gaia-theme": "gaia-components/gaia-theme",
+		"gaia-header": "gaia-components/gaia-header",
+		"gaia-button": "gaia-components/gaia-button",
+		"gaia-fonts": "gaia-components/gaia-fonts"
+	}
```

Comment:
My preference is to stick to tabs as I believe they are generally superior to spaces, and can be user-configured to any size. It is unfortunate that we have to use a hack for github, but it can be solved with a browser addon or customization.

Are folks ok with using a browser addon for this? Should just be a one-shot install then we can forget about it.